src/Entity/MollieGatewayConfigTranslation.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
```

Comment:
add enter after <?php in all files and comment 
/*
 * This file has been created by developers from BitBag.
 * Feel free to contact us once you face any issues or want to start
 * You can find more information about us on https://bitbag.io and write us
 * an email on hello@bitbag.io.
 */